### ISSUE TITLE
trying to compile getting  exit status 1

### ISSUE THREAD

**Issue by omryr** [ethereum/solc-js#43]
Hi i really need some help.. i have been trying to use solc to compile for the last 2 days, from console and from atom and i keep getting the error  " exit status 1 must provide file"
i have no idea what is going wrong, in the atom i built a contract and in the console i either gave a path to a contract or wrote one in  eth.compile.solidity("---contract---") and it just wont work..

any help would be greatly appericiated.

i am runing solc 0..3.6 btw

thanks for the help!

**Comment by axic** [comment]
Can you please explain the commands you're using?

**Comment by cesargalindo** [comment]
I'm also getting this error:    exit status 1 Must provide a file error 

Unable to compile.solidity this line in my javascript file:
```
  var compiled = web3.eth.compile.solidity("contract C{}");
   console.log(compiled);
```

ERROR:
ORIGINAL EXCEPTION: solc: exit status 1
Must provide a file

> solc --version
0.4.2+commit.af6afb04.mod.Emscripten.clang

Using:   ethereum:web3              0.15.3

**Comment by axic** [comment]
@omryr @cesargalindo solc-js is not compatible with web3.js

**Comment by cesargalindo** [comment]
Thanks @omryr, I figured that out earlier.

Here's how I got it working.  Here's what I did on my Mac.

1) Uninstalled Ethereum using Brew, I was using 1.5.x  unstable release.  Re-installed latest stable release 1.4.18

2) This still didn't fix it.  Uninstalled solidity:  
`npm uninstall solc -g`

3) Installed solidity using Brew, now it works:

```
brew install solidity
brew linkapps solidity

```

**Comment by axic** [comment]
Fixed by #101.
